src/commands/deploy.ts

```diff
@@ -177,13 +178,31 @@ export default class Deploy extends Command {
       this.log(chalk.white('Open up the url below in your browser:'));
       this.log();
 
+      const { project } = await this.got(
+        `v1/projects/${config.app}`
+      ).json<IDtailsProjectResponse>();
+
       const defaultSubdomain: string =
         config.region === 'iran' ? '.iran.liara.run' : '.liara.run';
       const urlLogMessage = DEV_MODE
         ? // tslint:disable-next-line: no-http-string
-          `    ${chalk.cyan(`http://${config.app}.liara.localhost`)}`
-        : `    ${chalk.cyan(`https://${config.app}${defaultSubdomain}`)}`;
-      this.log(urlLogMessage);
+          `    ${`http://${config.app}.liara.localhost`}`
+        : `    ${`https://${config.app}${defaultSubdomain}`}`;
+
+      if (project.defaultSubdomain) this.log(urlLogMessage);
```

Comment:
I meant something like this:

```suggestion
      if (project.defaultSubdomain && domains.length) this.log(urlLogMessage);
```